src/client/game/Player.js

```diff
@@ -132,3 +131,10 @@ Player.prototype.update = function() {
 Player.prototype.getSprite = function() {
   return this._sprite;
 }
+
+Player.prototype._sprint = function() {
+	if(this._WALK_SPEED == 250) {
```

Comment:
1. You should always use `===` unless you specifically are trying to use type conversion.
2. Instead of `if (this._WALK_SPEED === 250)` let's do `if (this._sprinting)`.
3. Please put a space in between the `if` statement and `(`.